azure-pipelines.yml

```diff
@@ -409,7 +409,6 @@ stages:
       - pwsh: |
           ./eng/generate-compiler-code.ps1 -test -configuration Release
         displayName: Generate Syntax Files
-        condition: or(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['compilerChange'], 'true'))
```

Comment:
This step takes ~30 seconds to complete and is not part of the longest job for CI. Hence conditionalizing this complicates our CI without making throughput any faster. Removing the conditional to simplify this path.